Shapes.java

```diff
@@ -1,25 +1,63 @@
-import java.util.*;
-
+import java.util.Scanner;
 public class Shapes{
-  public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
+  public static void main(String[] args){
+    Scanner in = new Scanner(System.in);
+
+    System.out.println("Enter number assigned to the shape you want:");
+    System.out.println("1.Rectangle");
+    System.out.println("2.Triangle");
+    System.out.println("3.Circle");
+    System.out.println("4.Trapezoid");
+    int shape = in.nextInt();
+
+    if (shape==1){
+      System.out.print("What is the width of the rectangle?");
+      int a = in.nextInt();
```

Comment:
Can you think of different names for your variables that would tell the reader what the variable represents?